test/parallel/test-https-server-close-idle.js

```diff
@@ -39,42 +39,45 @@ server.listen(0, function() {
   // Create a first request but never finish it
   const client1 = connect({ port, rejectUnauthorized: false });
 
-  client1.on('close', common.mustCall(() => {
-    client1Closed = true;
-  }));
+  client1.on('connect', common.mustCall(() => {
+    assert.strictEqual(connections, 1);
```

Comment:
```suggestion
```

It seems that the `'connect'` event can be emitted before the `'connection'` event on the server. I think we can remove this as the second connection is initiated from the listener of this event anyway.

Reply:
@santigimeno are you ok with this? I would like to see if CI is happy after this change.

Reply:
@lpinca sure thing. Just rebased and pushed with that change. Sorry for the delay. Thanks!